hazelcast/src/test/java/com/hazelcast/cp/internal/raft/impl/LocalRaftTest.java

```diff
@@ -491,6 +491,7 @@ public void when_leaderStaysInMinorityDuringSplit_thenItMergesBackSuccessfully()
         RaftEndpoint leaderEndpoint = group.waitUntilLeaderElected().getLocalMember();
 
         int[] split = group.createMajoritySplitIndexes(false);
+        Arrays.sort(split);
```

Comment:
Isn't the sort added also to the `split` method called below?

Reply:
Yes, in this case, the `sort` method is not needed, but I've added it here to explicitly show that the `split` array is sorted (because below there is a `binarySearch` call https://github.com/hazelcast/hazelcast/blob/09dc6aa1dc9424f1ec69fcadc8308915ed22d403/hazelcast/src/test/java/com/hazelcast/cp/internal/raft/impl/LocalRaftTest.java#L507)

The `sort` method is added to the `group.spilt(...)` method due to it is called from different places.

Reply:
I see, makes sense.